consumers/notifiers/smpp/notifier.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package smpp
+
+import (
+	"time"
+
+	"github.com/fiorix/go-smpp/smpp"
+	"github.com/fiorix/go-smpp/smpp/pdu/pdufield"
+	"github.com/fiorix/go-smpp/smpp/pdu/pdutext"
+	"github.com/mainflux/mainflux/consumers/notifiers"
+	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/mainflux/mainflux/pkg/transformers"
+	"github.com/mainflux/mainflux/pkg/transformers/json"
+)
+
+var _ notifiers.Notifier = (*notifier)(nil)
+
+type notifier struct {
+	t             *smpp.Transmitter
```

Comment:
I would consider calling this struct field `transmitter`, and one below `transformer`. When it comes to struct fields, I find it easier to read the code - example on one of the lines below `n.t.Submit()` does not tell me much as `n.transmitter.Submit()`.

@nmarcetic @dusanb94 opinions?

Reply:
For me, both are fine. **Transmitter** is a bit more self-explanatory, **t** is shorter. When reading the code, the one will probably need to check what it is in any case.

Reply:
Agree @drasko @dusanb94 , readability wins. I prefer to keep variable names short in scopes (like 1-3 letters), not struct fields where they should be obvious.